app/main.py

```diff
@@ -27,3 +27,7 @@ def ansewer_questions_brand_president():
             ],
         }
 
+@app.get("/{facet_name}/{facet_value}")
+def knowledge_panel(facet_name :Facetname,facet_value: Facetvalue ):
```

Comment:
why not just use a str for facet_value ?

Reply:
it is a str, U mean for facetname ?